db/migrate/20191213110357_add_columns_for_file.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsForFile < ActiveRecord::Migration[5.2]
```

Comment:
I think it would be great to introduce all this new database stuff with a single migration to keep the number of migrations small and easier to understand. 5 Migrations that all modify a newly introduced datatable aren't a good idea in my opinion.

- [ ] condense migration into a single one